### DISCUSSION TITLE
Django channels with subfolder option

### DISCUSSION THREAD

**Discussion by DanieleMilillo13** [django-tenants/django-tenants#788] [Q&A]
Hi, I have a problem with django tenants set in subfolder mode together with Django channels, every time I try to call an api "/r/demo/..." I get an error: `'FakeTenant' object has no attribute 'domain_subfolder'`.
What may I do to make things work together?
Thanks in advance

**Comment by smark-ARK** [comment]
Did someone figure this out?
Is there a way to do it??
